src/cljs/rems/application_list.cljs

```diff
@@ -90,19 +103,18 @@
            :todo (let [value (localize-todo (:application/todo app))]
                    {:value value
                     :td [:td.todo
-                         {:class (str (when (current-user-needs-to-do-something? app)
-                                        "text-highlight ")
-                                      (when (:application/past-deadline app)
-                                        "text-danger"))}
+                         {:class (when (current-user-needs-to-do-something? app)
```

Comment:
This is a nice change, I think it looks more clear now that only the :submitted column is highlighted.